src/ck-core/ck.C

```diff
@@ -2203,6 +2203,7 @@ void registerArrayMapProcNumExtCallback(int (*cb)(int, int, const int *)) {
 
 int CkMyPeHook() { return CkMyPe(); }
 int CkNumPesHook() { return CkNumPes(); }
+void CmiAbortHook(const char *msg) { CmiAbort(msg); }
```

Comment:
Might be better to do CmiAbort(“%s”, msg)

Reply:
I agree